engines/python/setup/djl_python/streaming_utils.py

```diff
@@ -23,10 +23,10 @@ def get_stream_generator(execution_engine: str):
         ## execution_engine passed to this function is not the same engine specified in serving.properties
         ## in djl-serving. For e.g Accelerate and neuronx use Python as the engine serving.properties
         ## The engine here refers to backend model parallel framework.
-        if execution_engine in {"DeepSpeed", "Accelerate"}:
+        if execution_engine in {
```

Comment:
should this be a list ["DeepSpeed", "Accelerate", "transformers-neuronx"] ?

Reply:
A set will be better?